packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,15 +2,18 @@
 
 ## Changelog
 
-### September 26, 2023
+### October 10, 2023
 
 #### New Features
 
+- **API-Based Message Extension**: We have introduced a whole new way to build search-based message extension - from a new API with Azure Functions or from your existing API with an OpenAPI description document. This is a great starting point for building a message extension that requires a backend to fetch data from a third-party API or a database.
```

Comment:
Shall we make it more clear that this release is supporting read-only API or GET only? as the last sentence is indicating so?

Reply:
Our UI indicates only `GET` API is supported, also there's going to be a blog announcement for API-ME so I didn't cover it in Changelog.

Reply:
that's ok as long as it is covered and clear on UI.